README.md

```
@@ -1,50 +1,94 @@
 
-This command will move the starter code to the **app-example** directory and create a blank **app** directory where you can start developing.
+-   **Scan the QR code** with the Expo Go app (Android) or Camera app (iOS).
+-   Press `a` to open in Android Emulator.
+-   Press `i` to open in iOS Simulator.
+-   Press `w` to open in Web browser.
 
-## Learn more
+## 📂 Project Structure
 
-To learn more about developing your project with Expo, look at the following resources:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)